client/blocks/daily-post-button/docs/example.jsx

```diff
@@ -0,0 +1,22 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import DailyPostButton from 'blocks/daily-post-button';
+import { dailyPromptPost } from 'blocks/daily-post-button/test/fixtures';
+
+const DailyPostButtonExample = () => {
+	return (
+		<div className="design-assets__group">
+			<DailyPostButton post={ dailyPromptPost } />
+		</div>
+	);
+};
+
+DailyPostButtonExample.displayName = 'DailyPostButton';
```

Comment:
It does get set automatically, but in this case we're setting it to a different name than the default one. This is so devdocs can index the example under the component name, rather than the example name.